crates/forge-runner/src/running.rs

```diff
@@ -199,7 +199,7 @@ pub(crate) struct ForkInfo {
 pub struct RunResultWithInfo {
     pub(crate) run_result: Result<RunResult, RunnerError>,
     pub(crate) fork_info: ForkInfo,
-    pub(crate) total_gas_used: f64,
+    pub(crate) total_gas_used: u128,
```

Comment:
Why, is there no float that is big enough? :c

Reply:
`calculate_tx_l1_gas_usage` (from blockifier) returns such a type and now we use it to calculate gas. I think this is the right approach (gas displayed in explorers is always an integer)

Reply:
If explorers do so then I cannot argue xD

Reply:
I mean, this is the way blockifier handles that (and we use functions from it). Can bring back those floats cause casting it to int is not mentioned in the official documentatnion